### DISCUSSION TITLE
Space Station Spaceframes?

### DISCUSSION THREAD

**Discussion by SMG001** [bcholmes/StarTrek2d20#116] [Q&A] [answered]
So, I don't know if this has been asked before, but since Utopia Planitia has released ways for us to create our own space stations, does this mean we're gonna get Space Station creation in the site too?

**Comment by bcholmes** [comment, marked as answer]
Yeah, there's an item in the issue list (#82) to support spaceframes. Just need the time to work on it.

**Reply by SMG001** [reply]
Ok, thanks for explaining
